### PR TITLE
Enable unit tests

### PR DESCRIPTION
**Pull request by JonasVautherin**
* Add unit tests for Action.
* Change the build setup. It should work by following these steps:

1. Clone this repo (for reviewing this PR, don't forget to checkout to this branch)
2. Run `bash fetch_archive_sdk.bash`. It will download the archive and unzip it in `bin/`
3. Open the project with Xcode. It should just work

### CODE REVIEW

**Comment by JonasVautherin** [comment]
@julianoes FYI, but no particular need for you to try it.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 xcuserdata/
 build/
 bin/
+
+.idea
```
Comment (julianoes) [inline .gitignore]: :smile:

File: Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Core.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+public class Core {
+    init() {
+        gRPC.initialize()
+    }
+    
+    public func connect() {
+        DispatchQueue.global(qos: .background).async {
+            print("Running backend in background (MAVLink port: 14540)")
+            runBackend(14540)
```
Comment (julianoes) [inline Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Core.swift]: Nice.

**Comment by ayameMBS** [comment]
I'have followed step 1 to 3 and that's fine. There is no issue. 
Then I have created my own project and drag and drop all the frameworks in it.
I do  "import Dronecode_SDK_Swift" in ViewController.swift and for example in viewDidLoad() I put "let action = Action()" 
and for this line, I got this issue : 'Action' initializer is inaccessible due to 'internal' protection level.
Something is wrong compare to what we have before.
Let's talk about it tomorrow

**Comment by JonasVautherin** [comment]
That's a small error in the code, I'm changing that, but it will come in the next PR. Merging this for now, as building the framework is working.

**Pull request merged**